tools/istioctl/plugin.yaml

```diff
@@ -3,13 +3,13 @@ tools:
   downloads:
     - name: istioctl
       downloads:
-        - os:
-            linux: linux
-            macos: osx
+        - os: macos
+          url: https://github.com/istio/istio/releases/download/${version}/istioctl-${version}-osx-amd64.tar.gz
```

Comment:
do we need to specify a cpu in this section in order for it to be applied correctly?

Reply:
in that case we get the missing platform exception - working on that separately